proto/allocation/allocation.proto

```diff
@@ -55,6 +55,12 @@ message AllocationRequest {
   // Metadata is optional custom metadata that is added to the game server at
   // allocation. You can use this to tell the server necessary session data
   MetaPatch metadata = 7;
+
+  // Ordered list of GameServer label selectors.
+	// If the first selector is not matched, the selection attempts the second selector, and so on.
```

Comment:
nit: indentation after the first comment line is incorrect.

Reply:
Done